docs/user_guide/mitigation.rst

```diff
@@ -74,6 +74,81 @@ Preprocessing
    
 .. currentmodule:: fairlearn.preprocessing
 
+Preprocessing algorithms transform the dataset to mitigate possible unfairness
+present in the data.
+Preprocessing algorithms in Fairlearn follow the `scikit-learn transformers
+<https://scikit-learn.org/stable/modules/generated/sklearn.base.TransformerMixin.html#sklearn.base.TransformerMixin>`_
+class, meaning that they can :code:`fit` to the dataset and :code:`transform` it
+(or :code:`fit_transform` to fit and transform in one go).
+
+.. _correlation_remover:
+
+Correlation Remover
+~~~~~~~~~~~~~~~~~~~
+Sensitive features can be correlated with non-sensitive features in the dataset.
+By applying the :code:`CorrelationRemover`, these correlations are filtered out while
```

Comment:
I am not super happy with the word "filtering" here, but haven't been able to come up with a good alternative... Thoughts? @romanlutz @rensoostenbach @LeJit

Reply:
Projected away? 😄 
@koaning may have thoughts as he came up with this 🙂

Reply:
"projected away" sounds fine 👍